### ISSUE TITLE
ZVV: de.schildbach.pte.exception.NotFoundException: https://online.fahrplan.zvv.ch/bin/mgate.exe

### ISSUE THREAD

**Issue by M0CHILER0** [schildbach/public-transport-enabler#607]
### Error
```
de.schildbach.pte.exception.NotFoundException: https://online.fahrplan.zvv.ch/bin/mgate.exe
de.schildbach.pte.util.HttpClient.getInputStream(HttpClient.java:296)
de.schildbach.pte.util.HttpClient.get(HttpClient.java:215)
de.schildbach.pte.AbstractHafasClientInterfaceProvider.jsonTripSearch(AbstractHafasClientInterfaceProvider.java:588)
```

**Comment by derhuerst** [comment]
I get `404` in [`hafas-client`](https://github.com/public-transport/hafas-client), too.

**Comment by derhuerst** [comment]
Duplicate of https://github.com/schildbach/public-transport-enabler/issues/606.

**Comment by schildbach** [comment]
Fixed via 81b1122204481bc980b091961244ecdb1a86aec0.
